### DISCUSSION TITLE
Is it possible to have multiple definitions in the same step?

### DISCUSSION THREAD

**Discussion by Thairam** [badeball/cypress-cucumber-preprocessor#1030] [Q&A]
In `.NET` using `specflow` is possible to have multiple definitions in the same step, like the code below:
```
[Then("a tabela '(.*)' da base de dados '(.*)' terá os valores na base de dados")]
[Then("a tabela '(.*)' da base de dados '(.*)' terá uma lista com os valores na base de dados")]
public void EntaoATabelaTeraUmaListaComOsValores(string tableName, string databaseName, Table specFlowTable)
{
  ...
}
```
Is it possible with the `cypress-cucumber-preprocessor`?

If yes, could you show me how?

**Comment by VigneshwaranSooriyamoorthy** [comment]
Regex is permitted, so maybe for this given example we can try something like below
`Then(/a tabela "([^\"]*)" da base de dados "([^\"]*)" terá (os valores na base de dados)|(uma lista com os valores na base de dados)/, () => {
});`
I can understand that this might be not that comfortable as what you are expecting, but a solution that may help :)
